flask/main.py

```diff
@@ -26,16 +26,16 @@ def decorated(*args, **kwargs):
     return decorated
 
 
-
 #Check if input term is part of a larger official term
 def isPartialTerm(search, filename):
     list_data = open(filename)
     for i in list_data:
+        print(search.upper(),str(i.rstrip("\n\r")).upper())
```

Comment:
Could you switch this to pything standard logging 
this looks like it would just be 
log.info("")

https://cloud.google.com/logging/docs/setup/python

It probably be helpful to add some more logging here too. 

Remember to add the google-cloud-logging to the requirements.

Thanks!